RNDateTimePicker.podspec

```diff
@@ -17,23 +15,8 @@ Pod::Spec.new do |s|
   s.source_files = "ios/**/*.{h,m,mm,cpp}"
   s.requires_arc = true
 
-  if fabric_enabled
-    folly_compiler_flags = '-DFOLLY_NO_CONFIG -DFOLLY_MOBILE=1 -DFOLLY_USE_LIBCPP=1 -Wno-comma -Wno-shorten-64-to-32'
-
-    s.compiler_flags = folly_compiler_flags + " -DRCT_NEW_ARCH_ENABLED=1"
-    s.pod_target_xcconfig = {
-      'HEADER_SEARCH_PATHS' => '"$(PODS_ROOT)/boost" "$(PODS_ROOT)/boost-for-react-native" "$(PODS_ROOT)/RCT-Folly"',
-      "CLANG_CXX_LANGUAGE_STANDARD" => "c++17"
-    }
-
-    s.dependency "React"
-    s.dependency "React-RCTFabric"
-    s.dependency "React-Codegen"
-    s.dependency "RCT-Folly"
-    s.dependency "RCTRequired"
-    s.dependency "RCTTypeSafety"
-    s.dependency "ReactCommon/turbomodule/core"
-
+  if ENV['RCT_NEW_ARCH_ENABLED'] == '1'
```

Comment:
Not sure you want to wrap this with  RCT_NEW_ARCH_ENABLED. I would not since the code already does that. They may add for various combos of RCT_NEW_ARCH_ENABLED.

Reply:
See https://reactnative.dev/docs/the-new-architecture/backward-compatibility-turbomodules

Reply:
good catch! I'll update it at some point / feel free to open a PR, thank you! :)